crates/rolldown_common/src/inner_bundler_options/types/target.rs

```diff
@@ -0,0 +1,68 @@
+// cspell:ignore Estarget
+use oxc::transformer::ESTarget as OxcEstarget;
+use std::str::FromStr;
+
+#[cfg(feature = "deserialize_bundler_options")]
+use schemars::JsonSchema;
+#[cfg(feature = "deserialize_bundler_options")]
+use serde::Deserialize;
+
+#[derive(Debug, Clone, Copy, Default)]
+#[cfg_attr(feature = "deserialize_bundler_options", derive(Deserialize, JsonSchema))]
+#[cfg_attr(feature = "deserialize_bundler_options", serde(rename_all = "camelCase"))]
+pub enum ESTarget {
+  ES5,
```

Comment:
NIT: `EsTarget`, `Es5` are more rust-style names. The std follows this style.

https://rust-lang.github.io/api-guidelines/naming.html#:~:text=In%20UpperCamelCase%2C%20acronyms%20and%20contractions%20of%20compound%20words%20count%20as%20one%20word%3A%20use%20Uuid%20rather%20than%20UUID%2C%20Usize%20rather%20than%20USize%20or%20Stdin%20rather%20than%20StdIn.

Reply:
I just copy it from oxc https://github.com/oxc-project/oxc/blob/b4258ee58e0a94773ccf85511530153025a58f4a/crates/oxc_transformer/src/options/es_target.rs#L7-L22